internal/backup/backup.go

```diff
@@ -277,7 +277,7 @@ func TriggerBackup(networkName string, backupData BackupData) func() {
 
 const tarPath = "modify-pghba.tar"
 
-func updatePghba(c dockerservice.Container, d dockerservice.Docker, content []byte) error {
+func updatePghba(c *dockerservice.Container, d dockerservice.Docker, content []byte) error {
```

Comment:
wondering why is it needs to change to a pointer variable. Because we update the `c` here https://github.com/spinup-host/spinup/pull/114/files#diff-cc73ad2e2e501df6e7a96f7724d68dda0da92408743c43c55428303f5b67e2ffR291?

Reply:
it was because `GetContainer` was updated to return a pointer, so this just changed to be consistent too.